### PR TITLE
APIGOV-27622 New "Manage your runtime security" section

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Enter a brief description of your changes to communicate to the reviewers what you changed and why.

## Deploy preview link

Please add the deploy preview link to the **specific page** that you've changed.

## Checklist for contributors

Before submitting this PR:

* Verify that all status checks have passed
* For more information on the Markdown linter rules, see [DavidAnson/markdownlint](https://github.com/DavidAnson/markdownlint)
* For first-time contributors, ensure that you have signed the [Axway CLA](https://cla.axway.com/)

### CODE REVIEW

**Comment by docsbuilder** [comment]
### <span aria-hidden="true">✅</span> Deploy preview is ready!

 |  Name | Link |
 |---------------------------------|------------------------|
 |<span aria-hidden="true">🔨</span> Latest commit | f58fef7 |
 |<span aria-hidden="true">😎</span> Deploy preview | http://PR-426.amplify-central.opendocs-builder.pcloud.axway.int |

File: content/en/docs/runtime_security/_index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Manage your runtime compliance
+linkTitle: Manage your runtime compliance
+weight: 495
+date: 2024-04-01
+hide_readingtime: true
+---
+
+With an increasing number of APIs, API providers and consumers of APIs, there is a pressing need to capture API traffic and scan the API traffic to assess the security threat(s).
```
Comment (lbadenhop) [inline content/en/docs/runtime_security/_index.md:9]:
```suggestion
With an increasing number of APIs, API providers and consumers of APIs, there is a pressing need to capture and scan API traffic to assess security threats.
```

File: content/en/docs/runtime_security/_index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+The Amplify API Management platform allows you to assess the runtime compliance of your APIs from a secruity threat perspective. This is known as API Runtime Compliance and has been integrated into the Amplify API Management. Runtime Compliance is an entitlement you can purchase from Axway as part of the API Management platform.
```
Comment (lbadenhop) [inline content/en/docs/runtime_security/_index.md:11]:
```suggestion
Amplify's Runtime Compliance entitlement for the API Management platform  allows you to assess the runtime compliance of your APIs from a security threat perspective. 
```

File: content/en/docs/runtime_security/_index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+## What is runtime compliance
+
+Runtime compliance is assessment of the potential security risk based on the real API traffic.  Amplify has integrated with Graylog API Security to monitor and assess the security threats from real captured API traffic.  For example, the captured API traffic is scanned by Graylog API security to detect real attacts, leaks, and other threats to your APIs.
+To get started, you have to create an Amplify environment which connects to Graylog API Security and to retrieve the security results.   The Graylog API Security data is correlated with the Managed APIs from the API Gateways connected to the Amplify Enterprise Marketplace to determine a security grading.   For APIs not found as a match to the API Gateways in the Amplify Enterprise Marketplace, these APIs are represented as Runtime Unmanaged APIs with a security grading.  These unmanaged runtime APIs are the group of unknown APIs not registed with an API Gateway.
```
Comment (lbadenhop) [inline content/en/docs/runtime_security/_index.md:16]:
```suggestion
To get started, must create an Amplify environment that connects to Graylog API Security and to retrieve the security results.   The Graylog API Security retrieves the security results and correlates the data with the Managed APIs from the API Gateways connected to the Amplify Enterprise Marketplace to determine a security grading.  

{{< alert title="Note" color="primary" >}} APIs that do not match the API Gateways in the Amplify Enterprise Marketplace are represented as Runtime Unmanaged APIs with a security grading.  These unmanaged runtime APIs are unknown APIs not registered with an API Gateway. See (link) for security grading details.{{< /alert >}} 
```

File: content/en/docs/runtime_security/_index.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+hide_readingtime: true
+---
+
+With an increasing number of APIs, API providers and consumers of APIs, there is a pressing need to capture API traffic and scan the API traffic to assess the security threat(s).
+
+The Amplify API Management platform allows you to assess the runtime compliance of your APIs from a secruity threat perspective. This is known as API Runtime Compliance and has been integrated into the Amplify API Management. Runtime Compliance is an entitlement you can purchase from Axway as part of the API Management platform.
+
+## What is runtime compliance
+
+Runtime compliance is assessment of the potential security risk based on the real API traffic.  Amplify has integrated with Graylog API Security to monitor and assess the security threats from real captured API traffic.  For example, the captured API traffic is scanned by Graylog API security to detect real attacts, leaks, and other threats to your APIs.
```
Comment (lbadenhop) [inline content/en/docs/runtime_security/_index.md:15]:
```suggestion
Runtime compliance is the assessment of potential security risks based on real-time API traffic.  Amplify has integrated with Graylog API Security to monitor and assess API traffic for security threats.  For example, the captured API traffic is scanned by Graylog API security to detect real attacks, leaks, and other threats.
```

**Review by robertengaxway — APPROVED** [review]

File: content/en/docs/runtime_security/_index.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+title: Manage your runtime compliance
+linkTitle: Manage your runtime compliance
+weight: 495
+date: 2024-04-01
+hide_readingtime: true
+---
+
+With an increasing number of APIs, API providers and consumers of APIs, there is a pressing need to capture and scan API traffic to assess security threats.
+
+Amplify's runtime compliance entitlement for the API Management platform allows you to assess the runtime compliance of your APIs from a security threat perspective. Runtime Compliance is an entitlement you can purchase from Axway as part of the API Management platform.
```
Comment (georgiana-caras) [inline content/en/docs/runtime_security/_index.md:11]: instead of entitlement i would say "feature" and remove this "for the API Management platform"

Comment (georgiana-caras) [reply content/en/docs/runtime_security/_index.md:11]: Runtime Compliance is an entitlement you can purchase from Axway as part of the API Management platform. 
I would replace this with: Contact Axway Support at support.axway.com or reach out to your Axway representative to learn how to enable the Runtime Compliance for your organization.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

File: content/en/docs/runtime_security/_index.md (hunk 5 above)
Comment (robertengaxway) [reply content/en/docs/runtime_security/_index.md]: Edited based on your feedback

**Pull request merged**